toplevel/opttoploop.ml

```diff
@@ -388,7 +390,7 @@ let execute_phrase print_outcome ppf phr =
               in
               Ophr_exception (exn, outv)
         in
-        !print_out_phrase ppf out_phr;
+          Log.log_itemf "phrases" log "%a" !print_out_phrase out_phr;
```

Comment:
If I understand correctly, the state at this commit is that most output goes to JSON here, but some lower-level errors of the toplevel (directive parsing, etc.) are sent directly to the underlying formatter -- potentially changing output order. This only affects `-json` users, so it is fine during a transitory period.